docs/release/release_0_4_13.md

```diff
@@ -139,6 +139,8 @@ Complete list of changes below:
 
 
 ## Bug Fixes
+- Update plugin docs index (#3964)
```

Comment:
We have them # in increasing order within a section - can these two get moved to end of section